torch/csrc/api/src/nn/init.cpp

```diff
@@ -36,6 +36,25 @@ struct Fan {
 };
 } // namespace
 
+TORCH_API double calculate_gain(Nonlinearity nonlinearity, double param) {
+  if (nonlinearity == Nonlinearity::linear ||
```

Comment:
You could add a switch statement here, considering `Nonlinearity` is an `enum`.

Reply:
If you throw an error for the default case, then the builds might pass. The builds failures seem to indicate that the end of the function has been reached without a return statement.

Reply:
I actually think it would be nicer if we just returned `1.0` in the default case. Then this huge block of comparisons can just be removed